Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "hunspell-sys"
 description = "Bindings to the hunspell C API."
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
Mind changing this back? I prefer to bump the version when I publish to crates.io.

Reply:
Done